src/containers/MediaElementContainer.test.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { renderWithRedux } from '../services/testing-helpers';
+import MediaElementContainer from './MediaElementContainer';
+import manifestAudio from '../json/mahler-symphony-audio';
+import manifestVideo from '../json/mahler-symphony-video';
+
+describe('MediaElementContainer component', () => {
+  test('renders successfully', () => {
```

Comment:
How is this test different than the next audio one on line 15?  Maybe this expectation could be included in the other two tests and this one can be removed or does it make sense to keep it separate?

Reply:
Yes, it's the same thing. I added this because it was the pattern followed in the other test suites. I guess there's no harm in removing this. I will remove this test and re-push :+1: